lib/membrane_opus/parser/frame_lengths.ex

```diff
@@ -0,0 +1,192 @@
+defmodule Membrane.Opus.Parser.FrameLengths do
```

Comment:
In this module, you seem to assume that you're going to get exactly a single packet each time, while in case of a self-delimiting stream, it most probably won't be true. A test for that would be useful too.

Reply:
can you give me an example of this scenario? i think i understand, but want to make sure.

Reply:
you may receive a stream like buffer{part of first packet}, buffer{another part of first packet}, buffer{last part of the first packet, entire second packet, part of third packet}, etc...

Reply:
gotcha. yeah that's going to require some state. will take some time to add